### ISSUE TITLE
Does minetest support simulated player joining/acting？

### ISSUE THREAD

**Issue by chrisliu1995** [minetest/minetest#13026]
Hi all, Is there any client API that can help to achieve the above functionality, like https://github.com/PrismarineJS/mineflayer?

**Comment by rubenwardy** [comment]
No, Minetest doesn't have an API for scriptable headless clients. This is very much a third party project thing, and does already exist as Hydra: https://github.com/dragonfireclient/hydra-dragonfire
